networking/configuring-cluster-wide-proxy.adoc

```diff
@@ -31,4 +36,28 @@ ifdef::openshift-rosa[]
 For information about standard installation prerequisites, see xref:../rosa_getting_started/rosa-aws-prereqs.adoc#prerequisites[AWS prerequisites for ROSA]. For information about the prerequisites for installation using AWS Security Token Service (STS), see xref:../rosa_getting_started_sts/rosa-sts-aws-prereqs.adoc#rosa-sts-aws-prerequisites[AWS prerequisites for ROSA with STS].
 endif::[]
 
+//OSDOCS-3290 Transparent Proxy Update
+[IMPORTANT]
+====
+The use of a proxy server to perform TLS re-encryption is currently not supported if the server is acting as a transparent forward proxy where it is not configured on-cluster via the `--http-proxy` or `--https-proxy` arguments.
+
+A transparent forward proxy intercepts the cluster’s traffic, but it is not actually configured on the cluster itself.
+====
+
 include::modules/cluster-wide-proxy.adoc[leveloffset=+1]
+include::modules/cluster-wide-proxy-updates.adoc[leveloffset=+1]
+
+//OSDOCS-2630 Proxy Configuration Expectations
+[NOTE]
+====
+You should not attempt to change the proxy or additional trust bundle configuration on the cluster itself. These changes should always be done via the command-line tools. Any changes that are made directly to the cluster will be reverted automatically.
+====
+
+//OSDOCS-2830 Customer Responsibilities
+[id="trust_bundle_responsibilities_cluster-wide-proxy-configuration"]
+=== Repsonsibilites for additional trust bundles
+
+If you supplied an additional trust bundle file, you are responsible for the following:
```

Comment:
This doesn't look like it is displaying the bullets properly in the live preview.